app/validators/bookmark_controller_validator.rb

```diff
@@ -1,7 +1,10 @@
 class BookmarkControllerValidator < ActiveModel::EachValidator
-  @@active_record_tables = ActiveRecord::Base.connection.tables.map(&:to_s)
   def validate_each(record, attribute, value)
-    controllers = ["dashboard"] + (@@active_record_tables + Permission.resources.map {|x| x.tableize }).uniq
+    controllers = ["dashboard"] + (self.class.active_record_tables + Permission.resources.map {|x| x.tableize }).uniq
```

Comment:
I didn't want to change the original behavior too much + the chance that the list will change in the future should be minimal, once the runtime is fully loaded (which was the issue originally, when the file could be loaded before the schema). Otherwise we would always need to go to the database for this, which are unnecessary 99.99999% times.